File: Ejercicio2.js
var readline = require('readline');
var rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});
meses=["Enero","Febrero","Marzo","Abril","Mayo","Junio","Agosto","Setiembre",
       "Octubre","Noviembre","Diciembre"]
while (true){
    var num = prompt(); 
    if (num>12 || num<1)
        break;
    else
        console.log(meses[num-1]);
}

File: ejercicio 2.py
meses = ['enero', 'ferero', 'marzo', 'abril', 'mayo', 'junio', 'julio', 'agosto', 'septiembre', 'octubre', 'noviembre', 'diciembre']

band = True
while band == True:
    mes = int(input('Ingrese el numero de un mes: '))

    if mes> 12:
       
        print('Error, numero ingresado incorrecto')

    else:
        
        if mes == 0:
            band = False
            print ('finalizar')

        else:
            
            mesesfull = mes - 1
            print(meses[mesesfull])
            

    


